Wavefront.SDK.CSharp/Wavefront.SDK.CSharp.csproj

```diff
@@ -4,7 +4,7 @@
     <PackOnBuild>true</PackOnBuild>
     <NoWarn>NU1701</NoWarn>
     <PackageId>Wavefront.SDK.CSharp</PackageId>
-    <PackageVersion>1.0</PackageVersion>
+    <PackageVersion>1.1.0-rc-001</PackageVersion>

```

Comment:
Should this just be 1.1.0?

Reply:
Planning to update to 1.1.0 right before release.  Packages get cached locally by version number, so it's easier to work with when each change has a different version number.